freshmaker/handlers/koji/rebuild_images_on_rpm_advisory_change.py

```diff
@@ -451,6 +452,29 @@ def _find_images_to_rebuild(self, errata_id, skip_nvrs=None):
             "Going to find all the container images to rebuild as " "result of %r update.",
             affected_nvrs,
         )
+
+        adv_is_compliance_priority = getattr(self.event.advisory, "is_compliance_priority", False)  
+        # We should only use a restriction for external repos for advisories that are not critical
+        # or important, even if they have the 'compliance-priority' label
+        restrict_to_ext_repos = None
```

Comment:
I feel "external repos" is a little confusing, I'd suggest just call it "compliance priority repositories".